velox/exec/tests/PlanNodeToStringTest.cpp

```diff
@@ -592,6 +592,20 @@ TEST_F(PlanNodeToStringTest, decimalConstant) {
       plan->toString(true));
 }
 
+TEST_F(PlanNodeToStringTest, integer) {
```

Comment:
This may not be the right place for the test. Let's add a test to DuckParserTest.